gradle.properties

```diff
@@ -1,2 +1,2 @@
 group=com.heapwhisperer.metrics.jvm
-version=1.0.2
+version=1.0.2-rocana2.0.0-SNAPSHOT
```

Comment:
Does this mean we have to make a release build of this dependency before we release 2.0.0?